src/sections/about-movement/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import Typography from "@/components/typography";
+import { useStaticQuery, graphql } from "gatsby";
+import Button from "@/components/button";
+
+import tech4covid19 from "@/assets/tech4covid19-logo.svg";
+
+import styles from "./index.module.css";
+
+const query = graphql`
+  query {
+    site {
+      siteMetadata {
+        authorUrl
+      }
+    }
+  }
+`;
+
+const AboutMovement = () => {
+  const data = useStaticQuery(query);
+  return (
+    <div className={styles.root}>
+      <div className={styles.container}>
+        <div className={styles.author}>
+          <a
+            href={data.site.siteMetadata.authorUrl}
+            className={styles.authorLink}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <img
+              className={styles.tech4covid19}
+              src={tech4covid19}
+              alt={data.site.siteMetadata.author}
+              title={data.site.siteMetadata.author}
+            />
+          </a>
+        </div>
+        <div>
+          <Typography
+            variant="h2"
+            weight="bold"
+            color="purple"
+            className={styles.title}
+          >
+            Sobre o movimento tech4COVID19
+          </Typography>
+        </div>
+
+        <div className={styles.videoContainer}>
+          <div className={styles.video}>
+            <iframe
+              src="https://www.youtube.com/embed/Dv5gaU5NaAA?rel=0"
+              frameBorder="0"
+              allow="autoplay; encrypted-media"
+              allowFullScreen
+              width="100%"
+              height="100%"
+              title="Sobre o movimento tech4COVID19"
+            />
+          </div>
+        </div>
+
+        <div className={styles.movementTextWrapper}>
+          <Typography
+            variant="smallBody"
```

Comment:
We can use `body` variant here, instead of set 20px on css file.

Reply:
Done!